### ISSUE TITLE
[Back] Bien plus tard, il faudra revoir l'import des signalements

### ISSUE THREAD

**Issue by hmeneuvrier** [MTES-MCT/histologe#1563]

**Comment by sfinx13** [comment]
Vu en point dev/UX c'est plus un sujet

**Comment by sfinx13** [comment]
En attente de fermeture @arnaudchouk

**Comment by arnaudchouk** [comment]
est ce que l import pourra etre revu avec la nouvelle fiche de signalement pro ?
je pensais qu on aurait quai plus de demandes mais il y en a toujours meme de territoires déjà ouverts et avec l api, et à terme la possibilité de créer des signalements à partir d'un partenaire connecté, il faudra potentiellement importer un lot de signalements déjà en cours de traitements chez partenaire non?

**Comment by sfinx13** [comment]
Oui, il me semble plus pertinent d'adapter l'import au nouveau formulaire pro. 

Et d'ailleurs avec un template téléchargeable directement depuis le BO, cela pourrait même devenir une fonctionnalité disponible directement depuis le BO. (on en a parlé entre nous)

Concernant l'API, dans un premier temps on ne prévoit pas la création de signalements via l'API. Ça pourra faire l'objet d'une évolution.

**Comment by arnaudchouk** [comment]
Ca marche
Pour la création de signalements à partir de l'API, vous pouvez noter que ce sera une évolution à faire avec une proba de + 90% :)
